atomate/vasp/firetasks/adsorption_tasks.py

```diff
@@ -325,10 +328,26 @@ def run_task(self, fw_spec):
                     vasptodb_kwargs={"parse_bader": False},
                     spec={"_category": _category},
                 )
+
+                static_dos_fws_params = self.get("static_dos_fws_params") or {}
+                static_dos_user_incar = static_dos_fws_params.get(
+                    "user_incar_settings", None
+                )
+                static_dos_user_kpts = static_dos_fws_params.get(
+                    "user_kpoints_settings", None
+                )
+
+                static_dos_incar_update = {"IVDW": 11, "GGA": "RP", "LAECHG": True}
+                for incar_k, incar_v in static_dos_user_incar.items():
+                    static_dos_incar_update[incar_k] = incar_v
                 static.tasks.insert(
-                    2,
-                    ModifyIncar(incar_update={"IVDW": 11, "GGA": "RP", "LAECHG": True}),
+                    2, ModifyIncar(incar_update=static_dos_incar_update)
                 )
+                if static_dos_user_kpts:
+                    static.tasks.insert(
+                        2, ModifyKpoints(kpoints_settings=static_dos_user_kpts)
```

Comment:
Should `kpoints_settings` be `kpoints_update`?

Reply:
Yes, looks like it! Good catch. I've tested the INCAR swaps but not the k-points here. Unfortunately, it looks like a deleted my fork of Atomate since I haven't used it since November. I'm going to close this PR, but feel free to copy/paste from here. Apologies for the inconvenience!